### PR TITLE
Added SharedPreference

### PR DESCRIPTION
**Pull request by yash-k9**

### CODE REVIEW

File: app/src/main/java/divyansh/tech/animeclassroom/common/Constants.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package divyansh.tech.animeclassroom.common
+
```
Comment (justdvnsh) [inline app/src/main/java/divyansh/tech/animeclassroom/common/Constants.kt]: Hi @yash-k9 . We do not need to create this extra constants file. One is already present in the utils package by the name C.kt
Try to use only that file for any types of Constants.

Comment (justdvnsh) [inline app/src/main/java/divyansh/tech/animeclassroom/common/Constants.kt]: Hi @yash-k9 . We do not need to create this extra constants file. One is already present in the utils package by the name C.kt
Try to use only that file for any types of Constants.

**Review by justdvnsh — CHANGES_REQUESTED** [review]
Hi @yash-k9. I have reviewed the file. Please make the required changes. Thank your for your contributions

**Comment by yash-k9** [comment]
Hi @justdvnsh, I've made the changes. please review

**Review by justdvnsh — APPROVED** [review]
Hi @yash-k9. Looks good to me . Thank you for your contributions

**Pull request merged**